darksidemanas.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "darksidemanas"
+project: "medical sos emergeny recordig and monitoring system"
+---
+
+# Project Name
+MEDICAL SOS EMERGENCY MONITORING AND RECORDING SYSTEM 
+## Summary
+
+I WANT TO MAKE IOT DEVICE TO CONNECT PATIENTS AND ALL MEDICAL ,SO THEY CAN MONITOR MULTIPLE PEOPLE AT THE SAME TIME IT WILL HELP IN SITUATIONS LIKE COVID-19.
+I WILL BE DEVEOPING ANDRIOD APP TO DO SO.
+
+## Plan
+
+1.FIRST WE NEED TO CONFIGURE RASPBERRY PI TO READ ANALOG SIGNALS USING A CUSTON DESINGED PCB 
+2.TO MAKE A ANDIOD APP WITH MULTIPLE LANGUAGE SUPPORT AN ALGORITHM TO LIST OUT PATIENTS WHO NEED MEDICAL ASSISTANCE ARGENTLY 
+3.PCB TOH CONNECT DIFFRENT MEDICAL PROBES WITH PI 
```

Comment:
```suggestion
3.PCB TOHCONNECT DIFFERENT MEDICAL PROBES WITH PI 
```

Reply:
Could you please elaborate on what you mean by medical probes? Are there any specific ones you're thinking of and do you already have access to them?

Reply:
medical probes suggest equipment like ECGs and other present in ICU or NICUS

Reply:
oh cool! could you please elaborate a little more on this within your plan (and add it to the file)? like how would you connect the equipment to your machine, what are you going to build in your circuit that will allow it to read the data from these machines? etc.

Reply:
like these machine uses analogue signals to transmit data to monitors and other equipment we will be using the mcb3008 and spi interface present in raspberry pi we will be reading analogue signals .

Reply:
ok cool! can you please add this in your file? also, is the budget inclusive of taxes and shipping?

Reply:
ok sure